### PR TITLE
Add http://mynode.local:3002 as a bitcoin block explorer

### PR DESCRIPTION
**Pull request by huey735**

### CODE REVIEW

**Comment by chimp1984** [comment]
I get a page not found

**Comment by huey735** [comment]
It is an implementation of [btc-rpc-explorer](https://github.com/janoside/btc-rpc-explorer) that can be run with the [Mynode project](http://mynodebtc.com/) locally.

**Comment by chimp1984** [comment]
Ah ok. I guess that might confuse other uses as well... Better might be to add a UI to enter a custom explorer or by prog arg.

**Comment by huey735** [comment]
I find it unlikely. My guess is that most people don't change the default and those that do, choose one that they're familiar with. Anyways, maybe I can add little disclaimer to the name field
`Mynode (requires dedicated node)`

**Comment by cd2357** [comment]
> Add http://mynode.local:3002 as a bitcoin block explorer

Or even https://custom-ip-or-hostname:3002

Some people run their own explorer on a raspi or some other host.

But that could be a future improvement.

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 1]
```diff
@@ -98,7 +98,8 @@
             new BlockChainExplorer("SoChain. Wow.", "https://chain.so/tx/BTC/", "https://chain.so/address/BTC/"),
             new BlockChainExplorer("Blockchain.info", "https://blockchain.info/tx/", "https://blockchain.info/address/"),
             new BlockChainExplorer("Insight", "https://insight.bitpay.com/tx/", "https://insight.bitpay.com/address/"),
-            new BlockChainExplorer("Blockchair", "https://blockchair.com/bitcoin/transaction/", "https://blockchair.com/bitcoin/address/")
+            new BlockChainExplorer("Blockchair", "https://blockchair.com/bitcoin/transaction/", "https://blockchair.com/bitcoin/address/"),
+            new BlockChainExplorer("Mynode.local", "http://mynode.local:3002/tx/", "http://mynode.local:3002/address/")
```
Comment (cd2357) [inline core/src/main/java/bisq/core/user/Preferences.java:102]: How about calling it "Local BTC RPC Explorer"?

"BTC RPC Explorer" is the official name of the explorer project: https://github.com/janoside/btc-rpc-explorer

Comment (cd2357) [reply core/src/main/java/bisq/core/user/Preferences.java:102]: But no, its fine, please disregard that, its only a cosmetic thing.

**Comment by cd2357** [comment]
utACK

**Comment by huey735** [comment]
@cd2357 
>Some people run their own explorer on a raspi or some other host.

@chimp1984 
>Ah ok. I guess that might confuse other uses as well... Better might be to add a UI to enter a custom explorer or by prog arg.

These two suggestions make sense together. Maybe it's worth to add custom field where the user would paste in the url because as @cd2357 said many people run their own block explorer. Be it btc-rpc-explorer, mempool.space, or other. 

But implementing that in the UI is beyond my skills.

**Comment by dcod3d** [comment]
There's already an open issue for using custom block explorer #4371 

But IMHO, a non public explorer should not be an option that's built into the app.

**Comment by cd2357** [comment]
> But IMHO, a non public explorer should not be an option that's built into the app.

Why not?

Edit: I mean, Bisq allows people to use their own (non-public) bitcoin node. I don't see why letting them use their own (non-public) blockchain explorer would be a problem.

So from my side: ACK 👍

**Comment by dcod3d** [comment]
I agree that bisq should allow (even encourage) use of an user's own blockchain explorer. However, hard coding in one company's implementation (explorer/url) isn't the right way to do it.

**Comment by sqrrm** [comment]
NACK

It doesn't make sense to hard code a link that doesn't work for most people. This should be either a command line argument and/or a text field where the specific explorer could be added.

**Comment by huey735** [comment]
Better implemented on https://github.com/bisq-network/bisq/pull/4733

**Pull request closed without merging**